examples/plot_qcbm_opt_history.py

```diff
@@ -35,12 +35,12 @@ def get_ordered_list_of_bitstrings(num_qubits):
 number_of_qubits = 4
 for step_id in data:
     step = data[step_id]
-    # if step["class"] == "get-initial-parameters":
+    # if step["stepName"] == "get-initial-parameters":
     #     number_of_qubits = int(
```

Comment:
This was commented as the worflowresult json in v1 didn't include the input parameters. I think one of the latest changed by Sabee added this capability so maybe its worth checking and updating?

Reply:
Okay, will do!

Reply:
Okay, I'll check with him on that today!

Reply:
Platform is still working on adding inputs to the workflowresult, so let's merge this PR and then we can update the logic for determining number of qubits later.